astronomer/providers/amazon/aws/sensors/emr.py

```diff
@@ -31,6 +32,23 @@ class EmrContainerSensorAsync(EmrContainerSensor):
         check query status on athena, defaults to 10
     """
 
+    def __init__(
+        self,
+        *,
+        poll_interval: float = 5,
+        **kwargs: Any,
+    ):
+        # TODO: Remove once deprecated
+        if poll_interval:
+            self.poke_interval = poll_interval
+            warnings.warn(
+                "Argument `poll_interval` is deprecated and will be removed "
+                "in a future release.  Please use  `poke_interval` instead.",
+                DeprecationWarning,
+                stacklevel=2,
+            )
+        super().__init__(**kwargs)
+
     def execute(self, context: Context) -> None:
```

Comment:
We should rename the usage of the `poll_interval` var to `poke_interval` and also pass `self.poke_interval` to it in this `execute` method.